pkg/util/trace/impl/motrace/report_log.go

```diff
@@ -103,6 +105,8 @@ func (m *MOZapLog) FillRow(ctx context.Context, row *table.Row) {
 	row.SetColumnVal(messageCol, table.StringField(m.Message))
 	row.SetColumnVal(extraCol, table.StringField(m.Extra))
 	row.SetColumnVal(stackCol, table.StringField(m.Stack))
+	row.SetColumnVal(statementIDCol, table.UuidField(m.StatementID[:]))
```

Comment:
should ignore `m.StatementID == [16]byte{}` case.
save `00000000-0000-0000-0000-000000000000` value is wast